src/Relations/ShuttleRelation.php

```diff
@@ -20,6 +20,30 @@ class ShuttleRelation extends ManyToManyRelation
     public $m_filterBox = false;
 
     /**
+     * Constructor.
+     *
+     * @param string $name The name of the relation.
+     * @param int $flags Flags for the relation
+     * @param string $destination The destination node (in module.name notation)
+     * @param string field for localKey
+     * @param string field for remoteKey
+     */
+
+     public function __construct($name, $flags = 0, $link, $destination, $local_key=null, $remote_keynull)
```

Comment:
sorry, my bad office closing and done in a hurry, yes should be =

El jul 12, 2016 12:20 PM, "Michele Rota" <notifications@github.com>
escribió:

> In src/Relations/ShuttleRelation.php
> <https://github.com/Sintattica/atk/pull/22#discussion_r70460619>:
>
> > @@ -20,6 +20,30 @@ class ShuttleRelation extends ManyToManyRelation
> >      public $m_filterBox = false;
> >
> >      /**
> > +     * Constructor.
> > +     *
> > +     * @param string $name The name of the relation.
> > +     * @param int $flags Flags for the relation
> > +     * @param string $destination The destination node (in module.name notation)
> > +     * @param string field for localKey
> > +     * @param string field for remoteKey
> > +     */
> > +
> > +     public function __construct($name, $flags = 0, $link, $destination, $local_key=null, $remote_keynull)
>
> shouldn't be $remote_key=null ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Sintattica/atk/pull/22/files/4fe9012ffc9bb7e9b92171c4d73d111989c92040#r70460619>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALYS2PvGl6g5OxW7YVZX-cuLX1P4dhdYks5qU7DbgaJpZM4JKf9k>
> .
>